GoogleForCreators/web-stories-wp
# Prevent media from scrolling on resize when tab is not selected

## Summary

Prevent media from scrolling on resize when tab is not selected.

## Testing Instructions

- Open the locally uploaded media tab.
- Select a different tab (eg. shapes or media3p).
- Open the network tab, and filter by 'media?context'.
- Resize the browser window.
- Expect no new pages of local media to be loaded in the network tab.

---



Fixes #3897
